src/main/java/frc/robot/constants/CompConstants.java

```diff
@@ -0,0 +1,14 @@
+package frc.robot.constants;
+
+/**
+ * CompConstants
+ */
+public class CompConstants {
+    private CompConstants(){
+        throw new IllegalStateException("Constants Class");
+    }
+    public static final boolean DEBUG_MODE = true;
```

Comment:
Should this use the kXXX naming convention?

Reply:
In our last workflow meeting, we came to consensus regarding the usage of SonarLint's code quality. We decided that we would follow all of their coding conventions so that we could have some consistant style in our code, one of those coding conventions include using all caps for constants (public static final fields) instead of the kXXX naming conventions we thought of before.